tests/PHPUnit/Framework/TestCase/UnitTestCase.php

```diff
@@ -18,16 +21,45 @@
  */
 abstract class UnitTestCase extends \PHPUnit_Framework_TestCase
```

Comment:
It "hurts" because 1 we wonder what does it do on the environment when we see it used, and 2 because it's a temptation to add code that doesn't belong there: the current state of this class is a good illustration. If we don't need it, we shouldn't have it. If it doesn't do anything, it has no reason to exist.